ros2topic/ros2topic/verb/pub.py

```diff
@@ -99,6 +99,9 @@ def set_msg_fields(msg, values):
                 raise SetFieldError(
                     '{field_name}.{e.field_name}'.format_map(locals()),
                     e.exception)
+        except ValueError as e:
+            raise SetFieldError(
+                '{field_name}'.format_map(locals()), e)
```

Comment:
Looking at the line again the whole use of `format_map` is pretty unnecessary. I used it for the field name concatenation in another line but here it only contains `field_name`. I will update it to use the literal variable instead...